packages/contracts/contracts/core/plugin/PluginUpgradeable.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.10;
+
+import "@openzeppelin/contracts-upgradeable/utils/introspection/ERC165Upgradeable.sol";
+
+import {DaoAuthorizableUpgradeable} from "../component/DaoAuthorizableUpgradeable.sol";
+import {IDAO} from "../IDAO.sol";
+
+/// @title PluginUpgradeable
+/// @notice An abstract, upgradeable contract to inherit from when creating a plugin being deployed via the transparent proxy pattern (see [OZ's article](https://blog.openzeppelin.com/the-transparent-proxy-pattern/)).
+abstract contract PluginUpgradeable is ERC165Upgradeable, DaoAuthorizableUpgradeable {
+    bytes4 public constant PLUGIN_INTERFACE_ID = type(PluginUpgradeable).interfaceId;
+
+    function __PluginUpgradeable_init(IDAO _dao) internal virtual onlyInitializing {
+        __DaoAuthorizableUpgradeable_init(_dao);
+    }
+
+    /// @notice adds a IERC165 to check whether contract supports PluginUpgradeable interface or not.
```

Comment:
```suggestion
    /// @notice adds an IERC165 to check whether contract supports the PluginUpgradeable interface or not.
```

Reply:
This is fixed here https://github.com/aragon/core/pull/91 so we can resolve this conversation.